index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Assignment 2</title>
+  <meta charset="utf-8">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="js/myscript.js"></script>
+</head>
+<body>
+<!-- Tony Mainelli
+     assignment02.html
+     INFO2340.WW
+     Thoendel
+     12/12/19
+-->
+<h1>The Wizard of OZ!</h1>
+
+<p>
+ <strong><span id="female1_01"></span></strong> lived in the midst of the great
```

Comment:
Great formatting! I like how all the span tags are easy to see and that you used numbering in the IDs to help keep them straight.